recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
-{% set version = "0.5.0" %}
+{% set version = "0.6.0" %}
 
 package:
   name: testpath
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/t/testpath/testpath-{{ version }}.tar.gz
-  sha256: 1acf7a0bcd3004ae8357409fc33751e16d37ccc650921da1094a86581ad1e417
+  sha256: 2f1b97e6442c02681ebe01bd84f531028a7caea1af3825000f52345c30285e0f
 
 build:
   number: 0
-  noarch: python
+  skip: True  # [py<36]
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
-    - flit >=3.2.0,<3.3
+    - flit >=3.2.0,<4
```

Comment:
I think there should be `flit-core` instead of `flit`, see https://github.com/jupyter/testpath/blob/de8ca59539eb23b9781e55848b7d2646c8c61df9/pyproject.toml#L2

Reply:
`flit` was replaced with `flit-core`.

https://github.com/AnacondaRecipes/testpath-feedstock/blob/pbs_0.6.0_20220224T100652/recipe/meta.yaml#L20&&L23 

As a result, there was an error with the `tomli` module. The solution for the issue was found in the following issue discussion:

`Missing module name tomli`
https://github.com/pypa/setuptools_scm/issues/608

by adding a version of `setuptools_scm` different than `6.1.1` the issue was resolved.